xhyve/xhyve.go

```diff
@@ -329,7 +330,16 @@ func (d *Driver) GetState() (state.State, error) {
 		return state.Stopped, nil
 	}
 
-	return state.Running, nil
+	psproc, err := ps.FindProcess(int(pid))
+	if err != nil {
+		return state.Error, err
+	}
+	// process name is truncated to 'docker-machine-d'
+	if strings.Contains(psproc.Executable(), "docker-machine") {
```

Comment:
nit: I like *successful return*(? sorry, I'm not good at English) to last `return` statement because of this projects other functions are almost same code style.
So, If you don't mind, could you change logic such as the following code?

```go
if !strings.Contains(psproc.Executable(), "docker-machine") {
	return state.Error, fmt.Errorf("Unable to find 'xhyve' process by PID: %d", pid)
}

return state.Running, nil
```